### DISCUSSION TITLE
error when running colmap2nerf.py script

### DISCUSSION THREAD

**Discussion by Reloof** [NVlabs/instant-ngp#999] [Q&A] [answered]
Windows10,I have added colmap's root folder which contains COLMAP.bat to my PATH and I can run colmap --help at terminal,but when i run`python ../instant-ngp/scripts/colmap2nerf.py --colmap_matcher exhaustive --run_colmap --aabb_scale 16`,The error is reported as follows
`running colmap with:
        db=colmap.db
        images="images"
        sparse=colmap_sparse
        text=colmap_text
warning! folders 'colmap_sparse' and 'colmap_text' will be deleted/replaced. continue? (Y/n)y
==== running: colmap feature_extractor --ImageReader.camera_model OPENCV --ImageReader.camera_params "" --SiftExtraction.estimate_affine_shape=true --SiftExtraction.domain_size_pooling=true --ImageReader.single_camera 1 --database_path colmap.db --image_path "images"
WARNING: Your current options use the maximum number of threads on the machine to extract features. Exracting SIFT features on the CPU can consume a lot of RAM per thread for large images. Consider reducing the maximum image size and/or the first octave or manually limit the number of extraction threads. Ignore this warning, if your machine has sufficient memory for the current settings.

==============================================================================
Feature extraction
==============================================================================

FATAL: command failed`

**Comment by Reloof** [comment, marked as answer]
Just reduce the resolution of the image.

**Reply by abubake** [reply]
How do you make it so it doesn't use the CPU but instead uses the GPU?

**Reply by toomy0toons** [reply]
https://github.com/NVlabs/instant-ngp/discussions/956
